include/cliffordsynthesis/Results.hpp

```diff
@@ -103,6 +130,8 @@ class Results {
   double runtime = 0.0;
   std::size_t solverCalls = 0U;
 
+  std::string pvals;
+  std::vector<std::vector<bool>> pvector;
```

Comment:
I am assuming the p here stands for permutation? Maybe just write this one out, so it is immediately clear.